File: src/main/Java/boj_step11/boj24267.java
import java.io.*;

public class boj24267 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        long n = Long.parseLong(reader.readLine());
        writer.append(String.valueOf(n * (n - 1) * (n - 2) / 6));
        writer.newLine();
        writer.append("3").close();
    }
}

File: src/main/Kotlin/boj_step11/boj24267.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine().toLong()
    w.write("${n * (n - 1) * (n - 2) / 6}\n3")
    w.close()
}